Resource.designer.cs does not yet support comma-delimited list for library %(Aliases) metadata

Context: https://github.com/xamarin/xamarin-android/issues/3808#issuecomment-599821699

In Visual Studio, I noticed that the description text for **Aliases** in the **Properties** window says:

```
A comma-delimited list of aliases to this reference.
```

I haven't yet found a mention of this in my quick searches of docs.microsoft.com, but it seems this is indeed allowed for the `%(Aliases)` metadata.

### Steps to reproduce

 1. Edit the `AndroidUpdateResourcesTest.CheckProjectReferenceAlias` test to replace:

    ```patch
    -Metadata = { { "Aliases", "Lib1" } },
    +Metadata = { { "Aliases", "Lib1A,Lib1B" } },
    ```

 2. Run the test.

### Expected behavior

The test passes.

It looks like the desired `external alias` lines to generate for _Resource.designer.cs_ would probably be:

```csharp
extern alias Lib1A;
extern alias Lib1B;
```

### Actual behavior

The test fails due to a syntax error in the generated _Resource.designer.cs_:

```csharp
extern alias Lib1A,Lib1B;
```

```text
Resources\Resource.designer.cs(1,20,1,21): error CS1002: ; expected
```

### Version information

xamarin-android/master@aaea55ff99ae033682dc8eef0817ddbe007aa542

--- a/src/Xamarin.Android.Build.Tasks/Tasks/GenerateResourceDesigner.cs
+++ b/src/Xamarin.Android.Build.Tasks/Tasks/GenerateResourceDesigner.cs
@@ -137,8 +137,10 @@ namespace Xamarin.Android.Tasks
 					assembly.CopyMetadataTo (item);
 					assemblies.Add (item);
 					string alias = assembly.GetMetadata ("Aliases");
-					if (!string.IsNullOrEmpty (alias))
-						aliases.Add (alias);
+					if (!string.IsNullOrEmpty (alias)) {
+						foreach (var a in alias.Split (new [] {','}, StringSplitOptions.RemoveEmptyEntries))
+							aliases.Add (a.Trim ());
+					}
 					Log.LogDebugMessage ("Scan assembly {0} for resource generator", fileName);
 				}
 				new ResourceDesignerImportGenerator (Namespace, resources, Log)

--- a/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs
+++ b/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs
@@ -26,7 +26,7 @@ namespace Xamarin.Android.Build.Tests
 			var proj = new XamarinAndroidApplicationProject () {
 				References = {
 					new BuildItem.ProjectReference (Path.Combine("..", library.ProjectName, Path.GetFileName (library.ProjectFilePath)), "Library1") {
-						Metadata = { { "Aliases", "Lib1" } },
+						Metadata = { { "Aliases", "Lib1, LibA1,LibB1" } },
 					},
 				},
 			};

--- a/src/Xamarin.Android.Build.Tasks/Utilities/ResourceDesignerImportGenerator.cs
+++ b/src/Xamarin.Android.Build.Tasks/Utilities/ResourceDesignerImportGenerator.cs
@@ -59,7 +59,8 @@ namespace Xamarin.Android.Tasks
 						var declaringTypeName = $"{reader.GetString (declaringType.Namespace)}.{reader.GetString (declaringType.Name)}";
 						if (declaringTypeName == resourceDesignerName) {
 							if (hasAlias) {
-								declaringTypeName = $"{alias}::{declaringTypeName}";
+								string [] aliases = alias.Split (new [] {','}, StringSplitOptions.RemoveEmptyEntries);
+								declaringTypeName = $"{aliases[0].Trim ()}::{declaringTypeName}";
 							}
 							CreateImportFor (declaringTypeName, typeDefinition, method, reader, hasAlias);
 						}

